graph/formats/rdf/testdata/index.html

```diff
@@ -5,7 +5,7 @@
     <meta content='width=device-width, initial-scale=1.0' name='viewport'>
     <link href='https://www.w3.org/StyleSheets/TR/base' rel='stylesheet' type='text/css'>
     <style>
-      body: {bacground-image: none;}
+      body: {background-image: none;}
```

Comment:
This can't be changed per the LICENSE of the test suite. Please revert all the changes in this directory.